src/lib/feedback-survey.js

```diff
@@ -0,0 +1,16 @@
+export const getIsUserCompletedFeedback = async(assessmentId, http) => {
```

Comment:
I think these two functions can just live in `src/components/results/feedback`, we try to keep lib with more common methods that might be used multiple places.